zone/lua_parser.cpp

```diff
@@ -1586,6 +1586,13 @@ int64 LuaParser::CalcSpellEffectValue_formula(Mob *self, uint32 formula, int64 b
 	return retval;
 }
 
+void LuaParser::RegisterBug(Client *self, BaseBugReportsRepository::BugReports bug, bool &ignoreDefault)
```

Comment:
Any reason to use the Base repository specifically?

Can we also use `snake_case` here?

Reply:
Done